src/Writer/XLSX/Options/PageSetup.php

```diff
@@ -6,8 +6,16 @@
 
 final class PageSetup
 {
+    public bool $fitToPage = false;
```

Comment:
```suggestion
    public readonly bool $fitToPage = false;
```

Reply:
sorry, this can't be accepted. The following error occurs:
```
$fitToPage cannot have default value
```
Besides, fitToPage value depends on fitToHeight and fitToWidth, which is passed to the constructor.

Reply:
Just remove the default value and set it in the constructor as

```php
$this->fitToPage = null !== $fitToHeight || null !== $fitToWidth;
```